torch/_torch_docs.py

```diff
@@ -736,11 +736,16 @@ def parse_kwargs(desc):
 
 add_docstr(torch.cat,
            r"""
-cat(seq, dim=0, out=None) -> Tensor
+cat(seq, dim=0, pad=False, pad_value=0, out=None) -> Tensor
 
 Concatenates the given sequence of :attr:`seq` tensors in the given dimension.
-All tensors must either have the same shape (except in the concatenating
-dimension) or be empty.
+Empty tensors will be ignored.
+If padding is set to false, all tensors must have the same shape
+(except in the concatenating dimension).
+If padding is set to true, all tensors do not need to have the same shape
```

Comment:
I don't know how to parse this.  There are two cases:
1) In the concatenating dimension: These clearly don't need to match, that's the point of padding
2) Not in the concatenating dimension: These never need to match, that's one of the points of cat.

So, should this sentence be here or is it equivalent to leave it off?